### ISSUE TITLE
Does it work on mobile?

### ISSUE THREAD

**Issue by Solsrngfan** [BuilderDolphin/dolphSol-Macro#202]
Because I don't know how it works

**Comment by b0b-depressed** [comment]
no it will not

**Comment by kuromian** [comment]
No, unless there's a mobile app that ports .ahk files to .apk apps that can run over other apps via the setting on android devices it can't. This is quite literally impossible unless someone tries hard enough to port it to mobile, which will end up taking unnecessary amounts of time and probably be super annoying to use anyways.
